deployments/examples/ocis_oc10_backend/ocis/config/identifier-registration.yaml

```diff
@@ -24,11 +24,11 @@ clients:
     insecure: yes
     trusted: yes
     redirect_uris:
-      - https://oc10.domain.com/apps/openidconnect/redirect/
-      - https://oc10.domain.com/apps/openidconnect/redirect
+      - http://oc10.example.org/apps/openidconnect/redirect
```

Comment:
for development purposes? if so add a comment that this must be removed in production instances!